components/benefit_sponsors/app/models/benefit_sponsors/serializers/csv/benefit_application_issuer.rb

```diff
@@ -0,0 +1,22 @@
+module BenefitSponsors
+  module Serializers
+    class BenefitApplicationIssuer
+      # employer_hbx_id,employer_feinlan_year_startlan_year_end,carrier_hbx_id,carrier_fein
+      def self.to_csv(benefit_application)
+        benefit_application.benefit_packages.inject([]) do |list, benefit_package|
+          benefit_package.sponsored_benefits.each do |sponsored_benefit|
+            sponsored_benefit.products(Date.today + 1.month).each do |product|
+              list.push [benefit_application.benefit_sponsorship.organization.hbx_id,
+                         benefit_application.benefit_sponsorship.organization.fein,
+                         benefit_application.effective_period.min.to_date,
+                         benefit_application.effective_period.max.to_date,
+                         product.issuer_profile.hbx_id,
+                         product.issuer_profile.fein].join(',')
```

Comment:
You really want to make sure these dates come out in something like ISO 8601  - does .to_date do that or do you need to strftime (asking because I think our custom locale formatting for dates does weird things, likes spits out  MM/DD/YYYY).

Reply:
Good catch, added `.strftime('%m/%d/%Y')` and updated the spec too.